src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Mapping/AddCohortCommandToDraftApprenticeshipDetailsMapperTests.cs

```diff
@@ -32,14 +31,13 @@ public async Task WhenMapping_ThenShouldSetProperties()
             Assert.AreEqual(fixture.TrainingProgramme, draftApprenticeshipDetails.TrainingProgramme);
         }
         
-        [TestCase(false, false)]
-        [TestCase(true, true)]
-        public async Task WhenReservationsIsEnabled_ThenShouldSetReservationId(bool isReservationsEnabled, bool expectReservationIdIsNotNull)
+        [Test]
+        public async Task WhenMapping_ThenShouldSetReservationId()
         {
-            var fixture = new AddCohortCommandToDraftApprenticeshipDetailsMapperTestsFixture().SetIsReservationsEnabled(isReservationsEnabled);
+            var fixture = new AddCohortCommandToDraftApprenticeshipDetailsMapperTestsFixture();
             var draftApprenticeshipDetails = await fixture.Map();
             
-            Assert.AreEqual(expectReservationIdIsNotNull ? fixture.Command.ReservationId : null, draftApprenticeshipDetails.ReservationId);
+            Assert.AreEqual(true ? fixture.Command.ReservationId : null, draftApprenticeshipDetails.ReservationId);
```

Comment:
This line should be re-written to remove the unnecessary ternary conditional operator.